desktop/linux/index.md

```diff
@@ -1,104 +1,172 @@
 ---
-description: Docker Desktop for Linux (Beta)
-keywords: docker, Desktop for linux, beta, tech preview
-title: Docker Desktop for Linux (Beta)
+description: Getting Started
```

Comment:
I can't leave a comment on images, so leaving a comment here;

- The `prefs.png` image has the `DockerCon 2022` menu item; which will be gone next week; can we create a screenshot without that menu item (so that we don't have to update the screenshot after next week)?
- The `changelog-placeholder.png` appears to be unused (unless I overlooked); can we remove it before it's merged?
- Same for `subscription-terms.png`
- Same for the `hello-world-nginx.png` image
- Some of the screenshots appear to be identical (or _very close_ to the "for mac" and/or "for windows" (for example, the "diagnostics" screenshot is _very_ similar  to https://docs.docker.com/desktop/mac/images/diagnose-support.png. This is of course the intent, and I was wondering if we should use the same images here (link to the existing ones)